File: Src/App/zabotalaboratory/zabotalaboratory.Analyses.Database/Entities/Talons.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace zabotalaboratory.Analyses.Database.Entities
{
    public class Talons
    {
        public virtual int Id { get; set; }

        [Required]
        [ForeignKey(nameof(AnalysesType))]
        public virtual int AnalysesTypeId { get; set; }
        [Required]
        public virtual AnalysesTypes AnalysesType { get; set; }

        public virtual List<AnalysesResult> AnalysesResult { get; set; }

        public virtual string PerformedBy { get; set; }

    }
}

File: Src/App/zabotalaboratory_client/src/app/shared/models/analyses/talons.ts
import { AnalysesResult } from './analyses-result';
import {AnalysesTypes} from "./analyses-types";

export class Talons{
  public id: number;
  public analysesType: AnalysesTypes;
  public analysesResult: AnalysesResult[];
  public performedBy: string;
}
